api/python/properties_prediction_d/mol_to_graph.py

```diff
@@ -0,0 +1,63 @@
+import dgl  # type: ignore
+import torch  # type: ignore
+import torch.nn.functional as F  # type: ignore
+
+from indigo import Indigo  # type: ignore
+
+indigo = Indigo()
+indigo.setOption("ignore-stereochemistry-errors", True)
+indigo.setOption("ignore-bad-valence", True)
+
+
+def featurize_atoms(mol):
+    feats = []
+    for atom in mol.iterateAtoms():
+        feats.append(atom.atomicNumber())
+    return {"atomic": F.one_hot(torch.tensor(feats), 60)}
+
+
+def featurize_edges(mol):
+    feats = []
+    for bond in mol.iterateBonds():
+        feats.append(float(bond.bondOrder()))
+    return {"ord": torch.tensor(feats * 2).reshape(-1, 1).float()}
+
+
+def featurize(mol, g, node_featurizer=None, edge_featurizer=None):
+    if node_featurizer is not None:
+        g.ndata.update(node_featurizer(mol))
+
+    if edge_featurizer is not None:
+        g.edata.update(edge_featurizer(mol))
+
+    return g
+
+
+def smiles_to_graph(smiles):
```

Comment:
Technically it is `mol_to_graph`, since `indigo.loadMolecule` automatically guesses input format and can load SMILES, InChI, Molfile, CML and couple of other formats.